packages/paywall/README.md

```diff
@@ -39,15 +39,27 @@ const provider = window.ethereum
 
 const paywall = new Paywall(networks)
 
-// set the configuration
-await paywall.setPaywallConfig(paywallConfig)
```

Comment:
This is not strictly required and passing it to `loadCheckoutModal` is cleaner imo.